packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/tests/useBlocksStore.test.js

```diff
@@ -1,25 +1,35 @@
 import * as React from 'react';
 
 import { lightTheme, ThemeProvider } from '@strapi/design-system';
-import { render, screen, renderHook } from '@testing-library/react';
+import { render, screen, renderHook, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import PropTypes from 'prop-types';
 import { IntlProvider } from 'react-intl';
 import { createEditor, Editor, Transforms } from 'slate';
 import { Slate, withReact } from 'slate-react';
 
+import { withLinks } from '../../plugins/withLinks';
 import { useBlocksStore } from '../useBlocksStore';
 
 const initialValue = [
   {
     type: 'paragraph',
-    children: [{ text: 'A line of text in a paragraph.' }],
+    children: [
+      {
+        type: 'link',
+        url: 'https://example.com',
+        children: [{ text: 'Some link' }],
+      },
+    ],
   },
 ];
 
+const user = userEvent.setup();
+
 const baseEditor = createEditor();
 
 const Wrapper = ({ children }) => {
-  const editor = React.useMemo(() => withReact(baseEditor), []);
+  const editor = React.useMemo(() => withReact(withLinks(baseEditor)), []);
```

Comment:
I just noticed we have a useMemo here but in our editor https://github.com/strapi/strapi/blob/e7e82d57df7e80015b9e78fb0ff8ea2d0656d049/packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js#L43C1-L44C1

we do not. Shouldn't we have the same setup in our tests?